### PR TITLE
MM-1598 Left and right arrow keys now scroll through pictures/files in preview mode

### PR DESCRIPTION
**Pull request by nickago**

### CODE REVIEW

**Comment by rgarmsen2295** [comment]
Looks good, +1

File: web/react/components/view_image.jsx [hunk 1]
```diff
@@ -5,113 +5,142 @@ var Client = require('../utils/client.jsx');
...
             function() {
-                $(self.refs.imageFooter.getDOMNode()).addClass("footer--show");
+                $(self.refs.imageFooter.getDOMNode()).addClass('footer--show');
             }, function() {
-                $(self.refs.imageFooter.getDOMNode()).removeClass("footer--show");
+                $(self.refs.imageFooter.getDOMNode()).removeClass('footer--show');
             }
         );
 
+        $(this.getDOMNode()).on('keyup', function(e) {
```
Comment (ralder) [inline web/react/components/view_image.jsx]: why does this bind event not with react jsx?

Comment (nickago) [inline web/react/components/view_image.jsx]: Our code base follows the convention of using JQuery for these types of event bindings, and this is consistent with that practice.

Comment (nickago) [inline web/react/components/view_image.jsx]: Thanks for the input. Right now our code base follows the convention of using JQuery for these types of event bindings, and this is consistent with that practice.

Comment (crspeller) [inline web/react/components/view_image.jsx]: Is there some reason for using JQuery? I think it would be best to use the react bindings whenever possible.

**Comment by nickago** [comment]
@crspeller This is ready for re-review

**Comment by rgarmsen2295** [comment]
+1

**Pull request merged**